doc/source/train/api/api.rst

```diff
@@ -163,6 +163,16 @@ Ray Train Output
 
     ~train.Result
 
+Ray Train Error
+---------------
+
+.. autosummary::
+    :nosignatures:
+    :template: autosummary/class_without_autosummary.rst
+    :toctree: doc/
+
+    ~train.error.SessionMisuseError
```

Comment:
Let's move `~train.base_trainer.TrainingFailedError` from the section below into this one as well?